data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
+            + "\n- [Environmental Advocacy with governments](https://www.conservation.org/priorities/working-with-governments)"
+            + "\n- [Fighting wildlife trafficking](https://www.conservation.org/priorities/wildlife-trade-and-trafficking)"
+            + "All [topics can be found online](https://www.conservation.org/priorities).",
+            updatedBy: "Alexander", updatedAt: "2020-02-16",
+        },
+        yearlyIncome: {
+            content: "- [149M $ in 2018](https://www.conservation.org/about/annual-report)" +
+                "\n- [158M $ in 2017](https://www.conservation.org/docs/default-source/publication-pdfs/fy17_annual_report.pdf?sfvrsn=e3d11069_3)" +
+                "\n- [212M $ in 2016](https://www.conservation.org/docs/default-source/publication-pdfs/2016-annual-report.pdf?sfvrsn=b268b345_3)" +
+                "\n- [165M $ in 2015](https://www.conservation.org/docs/default-source/publication-pdfs/2016-annual-report.pdf?sfvrsn=b268b345_3)",
+            updatedBy: "Alexander", updatedAt: "2019-02-16"
+        },
+        established: {
+            content: "[1987](https://www.conservation.org/about]",
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+        employees: {
+            content: "[53](https://www.conservation.org/docs/default-source/publication-pdfs/fy18_annual_report.pdf).",
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+    },
+    {
+        meta: {
+            name: "World Business Council for Sustainable Development",
+            link: "https://www.wbcsd.org/",
+            keywords: ["private-sector", "corporate", "network", "think-tank"],
+            label: "wbcsd",
+        },
+        description: {
+            content: "WBCSD is a network of 200 of the largest cooperations (8.5T $ combined revenue, 19M employees)."
+            + " It includes food multinationals as well as large petrochemical corporations."
+            + " Its stated mission is a sustainable economy until 2050. The economic interests of its membership make it seem unlikely that WBCSD can be a pivotal force in that endeavor."
+            + " WBCSD creates business cases on a multitude of topics to assist its members in becoming more sustainable and reducing the harm they inflict on Earth.",
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+        actions: {
+            content: "- Create business cases to boost sustainability ([Circular Economy](https://www.wbcsd.org/Programs/Circular-Economy), [Cities and Mobility](https://www.wbcsd.org/Programs/Cities-and-Mobility), [Climate & Energy](https://www.wbcsd.org/Programs/Climate-and-Energy), [Food & Nature](https://www.wbcsd.org/Programs/Food-and-Nature), [People](https://www.wbcsd.org/Programs/People) and [Redefining Value](https://www.wbcsd.org/Programs/Redefining-Value)."
```

Comment:
This needs more information. You just wrote down the topics.

Reply:
More would be better, I don't think that this less deep than others. Would you add a summary for each business case? What's missing

Reply:
Ok I am fine with it! :)